lesson-0-open-house/websockets-demo.yml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: websockets-demo
+  labels:
+    app: websockets-demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: websockets-demo
+  template:
+    metadata:
+      labels:
+        app: websockets-demo
+    spec:
+      containers:
+      - name: websockets-demo
+        image: quay.io/tomastomecek/websockets-demo:latest
```

Comment:
even though https://docs.openshift.com/container-platform/4.10/openshift_images/managing_images/image-pull-policy.html, you may want to explicitly set `imagePullPolicy: Always` in order to be adamant that your websockets-demo always breaks on the ImagePullBackoff during the presentation

Reply:
I actually haven't decided if I want to show the object definitions there - 20 minutes is not much time

but this is a very good idea to do for the first/second lesson